sdk/monitor/Azure.Monitor.OpenTelemetry.LiveMetrics/src/autorest.md

```diff
@@ -0,0 +1,17 @@
+# Generated code configuration
+
+Run `dotnet build /t:GenerateCode` to generate code.
+
+``` yaml
+input-file:
+    -  https://quickpulsespecs.blob.core.windows.net/specs/swagger-v2-for%20sdk%20only.json.
```

Comment:
is there a plan to change this to the Azure repo at a later date?

Reply:
Yes. This should point to rest api from Azure repo.